chadwick_bissette/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "chadwick_bissette",
+  "version": "1.0.0",
```

Comment:
Really at version 1.0.0?  If yes, go for it, but remember your semantic versioning.